src/js/tracing/nativeframes.ts

```diff
@@ -192,6 +192,17 @@ export class NativeFramesInstrumentation {
       },
     };
 
+    if (
+      measurements.frames_frozen.value <= 0 &&
+      measurements.frames_slow.value <= 0 &&
+      measurements.frames_total.value <= 0
+    ) {
```

Comment:
So we want to add the measurement when all the values are below zero?

Reply:
Yes, that's correct and that's as well the current behaviour of the other SDKs.

https://github.com/getsentry/sentry-cocoa/blob/bee0cc30381f47a3f82730c423a8cd14ebb6c483/Sources/Sentry/SentryFramesTracker.m#L359